corehq/util/models.py

```diff
@@ -218,7 +220,7 @@ class ComplaintBounceMeta(models.Model):
     destination = ArrayField(models.EmailField(), default=list, blank=True, null=True)
 
 
-class NullJsonField(JSONField):
+class NullJsonField(jsonfield_JSONField):
```

Comment:
Nit: `NullJsonField` suggests that the field stores `Null`. `DateField` stores a date, `CharField` stores characters, etc. `NullableJsonField` would be more appropriate.

Reply:
I don't disagree, but his code was moved from another place. Renaming the class feels out of scope for this PR.